docs/index.md

```diff
@@ -1313,25 +1318,163 @@ Statement 3 looks suspiciously like axiom `ax-2`, which states:
+unify them.
+The key feature we're going to use is that you can use the
+statement fragment selectors to *simultaneously* select two fragments,
+which may include *multiple* work variables, and
+then use "replace". Replace will use the two selected fragments, making
+this process *much* easier.
+
+> Use Alt+click to select, in step *4*, the last `->`.
+
+The statement fragment selector dialogue has appeared under step 4 and
+we now have this fragment selected:
+
+~~~~
+( ( &W3 -> &W4 ) -> ( &W3 -> &W2 ) )
+~~~~
+
+You could use the dialogue to
+increase or decrease the size of the fragment, but we don't need to.
+Now let's select the equivalent fragment in statement 3:
+
+> Use Alt+click to select, in step *3*, the third `->`.
+
+Another fragment selector dialogue has appeared under step 3 and
+it has this fragment selected:
+
+~~~~
+( ( ph -> ps ) -> ( ph -> ch ) )
+~~~~
+
+If you didn't pci the right fragment, use alt+click again to get the
+correct selection.
+
+**Note**: you *can* select two fragments at the same time.
+The ability to select two different fragments is important.
+
+Now we can use replacement:
+
+> Select "A with arrow" (replacement).
+> The replacement dialogue will appear, with our selections
+> entered as the "Replace what" and "Replace with" entries.
 
-We're getting close! Step 3 is proven, using ax-2.
+The two selected fragments have been copied into the fields.
+The earliest selected step is the "Replace what" and the
+later step is the "replace with"; in this case that's what we want.
+
+> Click on "Find subsitution". Notice that it shows a valid substitution
+> that replaces multiple work variables.
+
+Again, notice that "Replace what" doesn't need to be a single work variable.
+It can be an expression, one that even includes multiple work variables.
+If there are multiple work variables, and you apply the change, all will be
+replaced as necessary throughout the proof.
+
+> Click on "apply". Notice that multiple work variables have been replaced.
+
+We now have these two statements:
+
+~~~~
+4 P |- ( ( ph -> ( ps -> ch ) ) -> ( ( ph -> ps ) -> ( ph -> ch ) ) )
+3 P |- ( &W1 -> ( ( ph -> ps ) -> ( ph -> ch ) ) )
+~~~~
+
+Now let's do another replacement to make steps 3 and 4 even more similar.
+
+> Use Alt+click to select, in step *3*, the initial work variable &amp;W1
+> (only that work variable should be selected).
+> Use Alt+click to select, in step *4*, the first `->`.
+
+The second Alt+click highlighted this fragment in step 4:
+
+~~~~
+( ph -> ( ps -> ch ) )
+~~~~
+
+> Click on "Find substitution".
+
+Again the two fragments are copied in.
+However, in this case the order is the opposite of what we wanted,
+because we want to replace a work variable with an expression
+(not the other way around).
+The default is to use the earlier step first.
+In this case that's the opposite of what we wanted, so we'll
+swap them by pressing press the
+"up/down" (reverse) icon to he right of the "replace what" field.
```

Comment:
typo `swap them by pressing press the "up/down"` -> `swap them by pressing the "up/down"`